code/modules/cargo/blackmarket/blackmarket_items/consumables.dm

```diff
@@ -0,0 +1,63 @@
+/datum/blackmarket_item/consumable
+	category = "Consumables"
+
+/datum/blackmarket_item/consumable/clown_tears
```

Comment:
I don't suppose it's possible to add an item that contains berry juice as well?

It's not as black market-y as clown tears, but it'd be just as useful for bartenders.

Reply:
Sure, just typepath price and whatever. I made people write the descriptions and everything for me.